### PR TITLE
CTCP-2472: bump up dependencies

### PR DESCRIPTION
**Pull request by derekspalmer**

### CODE REVIEW

**Review by fmcgrath — DISMISSED** [review]
Seems OK.

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -2,13 +2,17 @@ import play.sbt.PlayImport._
...
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % bootStrapPlayVersion
+  )
 
   val test: Seq[ModuleID] = Seq(
-    "org.pegdown" % "pegdown" % "1.6.0" % "test",
-    "org.scalatest" %% "scalatest" % "3.0.5" % "test")
+    "org.pegdown"          % "pegdown"                % "1.6.0"  % "test",
+    "com.vladsch.flexmark" % "flexmark-all"           % "0.36.8" % "test",
+    "org.scalatest"       %% "scalatest"              % "3.2.9"  % "test",
+    "uk.gov.hmrc"         %% "bootstrap-test-play-28" % bootStrapPlayVersion
```
Comment (drnaylor) [inline project/AppDependencies.scala]: Needs to be suffixed with `% "test"`

Comment (derekspalmer) [reply project/AppDependencies.scala]: Yep - it does.
Will add.

**Review by fmcgrath — APPROVED** [review]
Seems OK.

**Review by drnaylor — APPROVED** [review]

**Comment by derekspalmer** [comment]
Bump up dependencies.

**Pull request merged**